integr8ly/grafana-operator
# add configuration for volumes and volumeMounts

## Description

- Allows users to append extraVolumes and volumeMounts to the Grafana deployment.

## Relevant issues/tickets

#367 #375

## Type of change




## Checklist


## Verification steps
- Create a new deployment using the example provided in `deploy/examples/extravolumes/grafana.yaml`
  - Or append an actual volume from your deployment
- Ensure the volumes and volumeMounts are parsed in their respective sections of the `grafana-deployment` resource along with other, native resources of those types
